third-party/cmake/BuildLibuv.cmake

```diff
@@ -41,18 +41,22 @@ set(UNIX_CFGCMD sh ${DEPS_BUILD_DIR}/src/libuv/autogen.sh &&
   CC=${DEPS_C_COMPILER})
 
 if(UNIX)
-  BuildLibUv(
+  BuildLibuv(
     CONFIGURE_COMMAND ${UNIX_CFGCMD}
-    INSTALL_COMMAND ${MAKE_PRG} install)
+    INSTALL_COMMAND ${MAKE_PRG} install
+                                CFLAGS+=-fno-omit-frame-pointer
+                                CFLAGS+=-fno-optimize-sibling-calls
+                                CFLAGS+=-fsanitize-memory-track-origins
+                                CFLAGS+=-fsanitize=memory)
```

Comment:
:+1:  currently this is just an experiment to see if the false-positive will go away.